File: exercicio15/ex15.java
public class ex15 {
    public static void main(String[] args) {
        String texto = "banana";
        char caracter = 'a';
        int contador = 0;

        for(int i = 0; i < texto.length(); i++){
            if(texto.charAt(i) == caracter){
                contador++;
            }
        }
        System.out.println("O texto " + texto + ", possui " + contador + " " + caracter);
    }
        }

File: exercicio15/ex15.js
function contadorCaracteres(texto, caracter) {
    let contador = 0;
    for (let i = 0; i < texto.length; i++) {
        if (texto[i] === caracter) {
            contador++;
        }
    }
  console.log(`o texto ${texto} possui ${contador} ${caracter}`);
}



contadorCaracteres('banana', 'a');
contadorCaracteres('banana', 'b');
contadorCaracteres('banana', 'n');